DEVELOPMENT.md

```diff
@@ -90,6 +90,30 @@ below.
 yarn run ibazel run tensorboard -- -- --logdir [LOG_DIR]
 ```
 
+#### Running TensorBoard on dev target with iBazel
+
+```shell
+(tf)$ ibazel run tensorboard:dev -- \
+--logdir gs://tensorboard-bench-logs \
+[--bind_all] \
+[--port 6008]
+```
+
+*   `ibazel`: File watcher for bazel; automatically rebuild and start the server
+    when files change.
+*   `:dev`: A target to bundle all dev assets with no vulcanization, which makes
+    the build faster
+*   `--logdir`: A folder where event file exists and TensorBoard read logs from.
+*   `--bind_all`: Used to view the running TensorBoard over the network than
+    from `localhost`, necessary when running at a remote desktop and accessing
+    the server from your local chrome browser.
+*   `--port`: followed by a port number to customize the port (the default port
+    is `6006`).
+
+Access your server at `http://<YOUR_SERVER_ADDRESS>:<PORT_NUMBER>/` if you are
+running TensorBoard at a remote desktop. Otherwise `localhost` should work.
+
+
 ### Debugging UI Tests Locally
```

Comment:
Can this title be updated to explain that this is for polymer tests (while the new section you added is for Angular webapp tests)